core/src/main/scala/org/apache/spark/deploy/SparkSubmitArguments.scala

```diff
@@ -566,7 +566,7 @@ private[deploy] class SparkSubmitArguments(args: Seq[String], env: Map[String, S
         |  --kill SUBMISSION_ID        If given, kills the driver specified.
         |  --status SUBMISSION_ID      If given, requests the status of the driver specified.
         |
-        | Spark standalone and Kubernetes only:
```

Comment:
It was added from https://github.com/apache/spark/pull/23518 @LucaCanali FYI

Reply:
@viirya

Reply:
Looks correct to me. `--total-executor-cores` is assigned to `spark.cores.max` which I don't see it is used in K8s. Maybe @dongjoon-hyun can confirm it.

Reply:
Yes, you are correct, @viirya . It seems that #23518 only checked the following code.
https://github.com/apache/spark/blob/8c11804c44066c059036cac8365f997fceb2c8e6/core/src/main/scala/org/apache/spark/deploy/SparkSubmit.scala#L677